Exercise_1.js

```diff
@@ -2,32 +2,149 @@
 // Write function have one paramiter(n). Return n numbers of Fibonacci number.
 // input: 6
 // output:1 1 2 3 5 8
+const Fibonacci1 = (a) => {
```

Comment:
name of function should start with verb. (example: findFibo....)
pleases set correct name of paramiter( ex: param,...)